.gitignore

```diff
@@ -44,6 +44,7 @@ token.txt
 /.wget-hsts
 .bash_history
 .byebug_history
+.solargraph.yml
```

Comment:
I don't know if including this was an accident, but it doesn't look related to this feature.  It's not a big deal and doesn't hurt anyone, and I've already put things in the .gitignore that probably should have been put in a personal global excludes file instead.